Structure_Engine/Create/Loads/GeometricalLineLoad.cs

```diff
@@ -57,6 +57,37 @@ public static GeometricalLineLoad GeometricalLineLoad(Line location, Loadcase lo
             };
         }
 
+        [Description("Creates a GeometricalLineLoad with a constant force across the length of the Line.")]
+        [InputFromProperty("location")]
+        [InputFromProperty("loadcase")]
+        [Input("force", "The force to be applied to the full length of the Line.", typeof(BH.oM.Quantities.Attributes.Force))]
+        [Input("panels", "The Panels to apply the GeometricalLineLoad to.")]
+        [Input("name", "The name of the created load.")]
+        [Output("geoLineLoad", "The created GeometricalLineLoad.")]
+        public static GeometricalLineLoad GeometricalLineLoad(Line location, Loadcase loadcase, Vector moment = null, Vector force = null, IEnumerable<Panel> panels = null, string name = "")
+        {
+            BHoMGroup<IAreaElement> objects = new BHoMGroup<IAreaElement>();
+            objects.Elements.AddRange(panels);
```

Comment:
```suggestion
            BHoMGroup<IAreaElement> group = new BHoMGroup<IAreaElement>();
            if (objects == null)
                group = null;
            else
                group.Elements = objects.ToList();
```

This follows a similar trend to the other loads, and ensures that the `group` is never `null` (which is causing the error in the Robot_Toolkit) - see:
https://github.com/BHoM/Robot_Toolkit/blob/aafb6067b31db673ca3a08c0ea7e08bb07ee2b3e/Robot_Adapter/CRUD/Create/Loads/Loads.cs#L97-L99